src/frontend/src/pages/ChangeRequestsPage/ChangeRequestsTable.tsx

```diff
@@ -152,7 +161,15 @@ const ChangeRequestsTable: React.FC = () => {
         onPageSizeChange={(newPageSize) => setPageSize(newPageSize)}
         loading={isLoading}
         error={error}
-        rows={data || []}
+        rows={
+          // flatten some complex data to allow MUI to sort/filter yet preserve the original data being available to the front-end
+          data?.map((v) => ({
+            ...v,
+            carNumber: v.wbsNum.carNumber,
+            submitter: fullNamePipe(v.submitter),
+            reviewer: fullNamePipe(v.reviewer)
+          })) || []
```

Comment:
what's the benefit of doing it this way instead of using the valueFormatter?

Reply:
OK so there's big problems with using value formatter. Value for matter, preserves movies, handling of the original data type, an object when a user input six string to filter the column, movie compares the string to the original type, an object. We have yet to figure out how to properly get the input string to be properly compared to the original object people saw so instead, we are just going to flatten the data before it gets to MUI, but after it comes in from the API. Does provides the most amount of flexibility and fixes the problem in the easiest way possible